plugins/armorblox/icon_armorblox/triggers/get_incidents/schema.py

```diff
@@ -0,0 +1,329 @@
+# GENERATED BY KOMAND SDK - DO NOT EDIT
+import insightconnect_plugin_runtime
+import json
+
+
+class Component:
+    DESCRIPTION = "Get a list of incidents identified by Armorblox, by default it starts polling all the incidents since last day"
+
+
+class Input:
+    
+    INTERVAL = "interval"
+    
+
+class Output:
+    
+    INCIDENTS = "incidents"
+    
+
+class GetIncidentsInput(insightconnect_plugin_runtime.Input):
+    schema = json.loads("""
+   {
+  "type": "object",
+  "title": "Variables",
+  "properties": {
+    "interval": {
+      "type": "integer",
+      "title": "Fetch_Interval",
+      "description": "Polling inteval in seconds",
+      "default": 600,
```

Comment:
Is there a way to set a minimum here so that people don't query every second for e.g.?

Reply:
Yes Rajat, we can set these values. Could you please suggest the minimum value to set?

Reply:
Use 600 for minimum too for now.

Reply:
Okay sure

Reply:
We cannot mention a keyword as Minimum. In the default/example we can set the values. I am setting the values as '600' only as of now...